src/index.tsx

```diff
@@ -4,21 +4,42 @@ import { QueryClient, QueryClientProvider } from "react-query";
 import { ReactQueryDevtools } from "react-query/devtools";
 import React from "react";
 import ReactDOM from "react-dom";
-import { BrowserRouter as Router } from "react-router-dom";
+import { Auth0Provider } from "@auth0/auth0-react";
+import { createBrowserHistory } from "history";
+import { getConfig } from "module/auth";
 
 import * as app from "component/app";
 import reportWebVitals from "reportWebVitals";
 
+// export default createBrowserHistory();
+const history = createBrowserHistory();
+const config = getConfig();
+
+//how do we set this up?
+const onRedirectCallback = (appState: any) => {
+  history.push(
+    appState && appState.returnTo ? appState.returnTo : window.location.pathname
+  );
+};
+
+const providerConfig = {
+  domain: config.domain,
+  clientId: config.clientId,
+  ...(config.audience ? { audience: config.audience } : null),
+  redirectUri: "http://localhost:3006/profile",
```

Comment:
I think we should follow the configuration mechanism we already have. Let's not hard code magic strings like that.